target/cheri-common/cheri-helper-utils.h

```diff
@@ -170,6 +170,8 @@ static inline void cheri_update_pcc_for_exc_return(cap_register_t *pcc,
                                                    cap_register_t *src_cap,
                                                    target_ulong new_cursor)
 {
+    assert(cap_has_perms(pcc, CAP_ACCESS_SYS_REGS) &&
```

Comment:
Morello allows it, it just tag clears the target... (ew)

Reply:
This helper function is only used by MIPS and RISC-V, so okay to keep the assertion?

Reply:
Then please add a comment making that clear so someone doesn't come along and refactor Morello to use the helper without tweaking the assert